src/test.sol

```diff
@@ -69,6 +69,20 @@ contract DSTest {
         }
     }
 
+    function assertFalse(bool condition) internal {
+        if (condition) {
+            emit log("Error: Assertion Failed");
+            fail();
+        }
+    }
```

Comment:
Couldn't this simply call `assertTrue` like this?

```solidity
function assertFalse(bool condition) internal {
    assertTrue(!condition);
}
```

Reply:
Yes we can also do this !